src/plugins/management/public/types.ts

```diff
@@ -70,9 +69,6 @@ export interface ManagementAppMountParams {
   element: HTMLElement; // element the section should render into
   setBreadcrumbs: (crumbs: ChromeBreadcrumb[]) => void;
   history: ScopedHistory;
-  theme: ThemeServiceStart;
-  /** @deprecated - use `theme` **/
-  theme$: Observable<CoreTheme>;
```

Comment:
Sorry about the back-and-forth on this. In my previous PR, I thought it would be convenient to provide `theme` in `ManagementAppMountParams` and deprecated `theme$`. While working through this PR I found that the `coreStart` contract is usually available in all the places ManagementAppMountParams is available. I decided to remove `theme` from being passed through management app mount params , as it's better to get `theme` (and i18n) from coreStart.